.github/workflows/build.yaml

```diff
@@ -31,20 +31,12 @@ defaults:
     shell: bash
 
 jobs:
-  d:
-    name: Discover Platforms
-    runs-on: ubuntu-latest
-    outputs:
-      missing-platforms: ${{steps.discover.outputs.missing-platforms}}
-      p: ${{steps.discover.outputs.platforms}}
-    steps:
-      - name: Discover Platforms
-        id: discover
-        uses: AntelopeIO/discover-platforms-action@v1
-        with:
-          platform-file: .cicd/platforms.json
-          password: ${{secrets.GITHUB_TOKEN}}
-          package-name: builders
+  build-base:
```

Comment:
Since this job (that uses the reusable workflow) doesn't have a name it seems to show up as simply `build-base` in the GUI in various places. I'm a little surprises it doesn't inherit the callee workflow name. Maybe we could consider giving this a name (even just `Build`) to show up nicer.

![Screenshot 2023-07-13 at 22-22-58 PH Create workflow to test backward compatibility of performance harness · AntelopeIO_leap@b59fede](https://github.com/AntelopeIO/leap/assets/32485495/322fc5ba-5666-47ac-9013-c610ba6b2fb8)

---

![Screenshot 2023-07-13 at 22-23-21 AntelopeIO_leap C implementation of the Antelope protocol](https://github.com/AntelopeIO/leap/assets/32485495/3247486c-a234-4981-9ff1-118cecf66d35)

Reply:
Updated with name `Run Build Workflow`:
![image](https://github.com/AntelopeIO/leap/assets/11048242/95dce4bd-a23f-4a0c-921a-f2a1431ff704)
![image](https://github.com/AntelopeIO/leap/assets/11048242/c6271936-61de-492a-b28e-75f39298eace)